src/test/java/iudx/resource/server/authenticator/JwtAuthServiceImplTest.java

```diff
@@ -262,8 +276,24 @@ public void success4ConsumerTokenEntitiesAPI(VertxTestContext testContext) {
     jwtData.setIat(1627408865);
     jwtData.setIid("rg:example.com/79e7bfa62fad6c765bac69154c2f24c94c95220a/resource-group");
     jwtData.setRole("provider");
-    jwtData.setCons(new JsonObject().put("access", new JsonArray().add("api")));
-
+    JsonObject access = new JsonObject()
+            .put("api", new JsonObject().put("limit", 1000).put("unit", "number"))
+            .put("sub", new JsonObject().put("limit", 10000).put("unit", "MB"))
+            .put("file", new JsonObject().put("limit", 1000).put("unit", "number"))
+            .put("async", new JsonObject().put("limit", 10000).put("unit", 122));
```

Comment:
Instead of creating the same access JSON in every test, it would be better to have a method that generates and returns the JSON. This approach helps avoid code duplication.

Reply:
also check unit for async